src/FlanCommand.ts

```diff
@@ -95,14 +95,17 @@ export default abstract class FlanCommand extends Command {
       ),
       database: {
```

Comment:
I'm thinking that we should just remove all the config for db

Reply:
All of it would get overridden by env vars or  the `flan.config.json` anyway right? I think we could remove it.

Reply:
lets think about it a bit more, I might want to remove the need for the config file altogether